x/evm/types/types.go

```diff
@@ -1285,3 +1287,20 @@ func CommandIDsToStrings(commandIDs []CommandID) []string {
 
 	return commandList
 }
+
+// ValidateCommandQueueState checks if the keys of the given map have the correct format to be imported as command queue state.
+// The expected format is {block height}_{[a-zA-Z0-9]+}
+func ValidateCommandQueueState(state map[string]codec.ProtoMarshaler) error {
+	for key := range state {
+		keyParticles := strings.Split(key, utils.DefaultDelimiter)
+		if len(keyParticles) != 2 {
+			return fmt.Errorf("expected key %s to consist of two parts", key)
+		}
+
+		if _, err := strconv.ParseInt(keyParticles[0], 10, 64); err != nil {
+			return fmt.Errorf("expected first key part of %s to be a block height", key)
+		}
```

Comment:
is this not going to fail with a burn command in the queue?

Reply:
No it shouldn't. Burn commands' block heights at here are all 0's.